Tools/CISupport/ews-build/steps.py

```diff
@@ -1860,10 +1860,19 @@ def run(self):
         remote = self.getProperty('remote', DEFAULT_REMOTE)
         validators = self.VALIDATORS_FOR.get(remote, [])
         lower_case_reviewers = [reviewer.lower() for reviewer in reviewers]
-        if validators and not any([validator.lower() in lower_case_reviewers for validator in validators]):
+        lower_case_validators = [validator.lower() for validator in validators]
+        if validators and not any([validator in lower_case_reviewers for validator in lower_case_validators]):
             defer.returnValue(self.fail_build_due_to_no_validators(validators))
             return
 
+        # Validators are a special case, not all validators are reviewers. If we have a reviewer that is a validator
+        # but NOT a reviewer, remove them
```

Comment:
"If we have a reviewer that is a validator but NOT a reviewer" reads very weird, a reviewer which is not a reviewer 🤔 

can frame it something like: "If we have a PR reviewer that is a validator but NOT a WebKit Reviewer"